html/changelogs/XeonFaux - HOG.yml

```diff
@@ -0,0 +1,6 @@
+author: XeonFaux
+
+delete-after: True
+
+changes: 
+  - rscadd: "Hand of God: Coming to a gamemode near you!"
```

Comment:
Very descriptive and unhelpful. What's the gamemode about, how will this effect existing mechanics, when will it be playable. Is this just a framework that's unplayable - if so, why does this matter to the players? Changelogs should ideally be informative and let the players know exactly as to what's changed and how it effects them.